### PR TITLE
Introduce `CrossBuildSession` scope

### PR DESCRIPTION
**Pull request by alllex**
...because there is an unscoped service registry for the `CrossBuildSessionState`.

### CODE REVIEW

**Review by adammurdoch — APPROVED** [review]

**Comment by alllex** [comment]
@bot-gradle test without pts

File: platforms/core-runtime/base-annotations/src/main/java/org/gradle/internal/service/scopes/Scope.java [hunk 1]
```diff
@@ -36,17 +36,22 @@ interface Global extends Scope {}
      */
     interface UserHome extends Global {}
 
+    /**
+     * TBD
```
Comment (tresat) [inline platforms/core-runtime/base-annotations/src/main/java/org/gradle/internal/service/scopes/Scope.java:40]: 💅 TBD?

**Review by tresat — APPROVED** [review]

**Comment by alllex** [comment]
@bot-gradle test

File: platforms/core-runtime/base-annotations/src/main/java/org/gradle/internal/service/scopes/Scope.java (hunk 1 above)
Comment (alllex) [reply platforms/core-runtime/base-annotations/src/main/java/org/gradle/internal/service/scopes/Scope.java]: Done

**Pull request merged**